opus/application/apps/ui/views.py

```diff
@@ -105,12 +107,31 @@ def api_last_blog_update(request):
             lastupdate = fp.read().strip()
     except:
         try:
-            log.error('api_last_blog_update: Failed to read file "%s"',
+            log.error('api_notifications: Failed to read file "%s"',
                       settings.OPUS_LAST_BLOG_UPDATE_FILE)
         except:
-            log.error('api_last_blog_update: Failed to read file UNKNOWN')
+            log.error('api_notifications: Failed to read file UNKNOWN')
 
-    ret = json_response({'lastupdate': lastupdate})
+    notification = None
+    notification_modify = None
+    try:
+        with open(settings.OPUS_NOTIFICATION_FILE, 'r') as fp:
+            notification = fp.read().strip()
```

Comment:
I think it would be good to make sure that a completely empty file, and a non-existent file, look the same to an outside user. So perhaps a line after this that says:
    if not notification: notification = None
(this will catch empty strings)
Could do the same for the blog date above to be consistent.